types/content.go

```diff
@@ -39,10 +40,13 @@ type RulePluginInfo struct {
 
 // RuleErrorKeyContent wraps content of a single error key.
 type RuleErrorKeyContent struct {
-	Generic   string           `json:"generic"`
-	Metadata  ErrorKeyMetadata `json:"metadata"`
-	TotalRisk int              `json:"total_risk"`
-	Reason    string           `json:"reason"`
+	Metadata   ErrorKeyMetadata `json:"metadata"`
+	TotalRisk  int              `json:"total_risk"`
```

Comment:
I'm not clear about this one. Why is it here?

Reply:
What do you mean? TotalRisk?

Reply:
it was always there, i just moved the field:)